main.tf

```diff
@@ -25,6 +25,29 @@ variable "node_image" {
   #default = "CentOS7.8" #-OpenHPC"
 }
 
+resource "openstack_networking_secgroup_v2" "secgroup_slurm_login" {
+  name        = "secgroup_slurm_login"
+  description = "Rules for the slurm login node"
+  # Fully manage with terraform
+  delete_default_rules = true
+}
+
+resource "openstack_networking_secgroup_rule_v2" "secgroup_slurm_login_rule_egress_v4" {
+  direction         = "egress"
+  ethertype         = "IPv4"
+  security_group_id = openstack_networking_secgroup_v2.secgroup_slurm_login.id
+}
+
+resource "openstack_networking_secgroup_rule_v2" "secgroup_slurm_login_rule_ingress_tcp_v4" {
+  direction         = "ingress"
+  ethertype         = "IPv4"
+  # FIXME: only allow ports needed
```

Comment:
Just to clarify, we need to open up the port for slurmdbd.

Reply:
Looks like 3306 from [docs](https://slurm.schedmd.com/slurmdbd.conf.html)